File: c/Lista2/Desafio4.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){
    
    float x, Cx;

printf("Digite o número: ");
scanf("%f", &x);

Cx = pow (x,3);
printf("O número elevado ao cubo é: %3.0f",Cx);

}

File: js/lista2/Desafio4.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
}); 

/*
Desafio4- Faça um algoritmo para ler uma variável:

Apresentar seu cubo.
*/

rl.question('Digite um número: ', (numero) => {
    let cubo = Math.pow(numero,3);
        
        console.log(`O cubo do número: ${numero} é: ${cubo}`);
      rl.close();
  });